pylint/config/argument.py

```diff
@@ -199,3 +211,32 @@ def __init__(
         # argparse uses % formatting on help strings, so a % needs to be escaped
         self.help = arg_help.replace("%", "%%")
         """The description of the argument."""
+
+
+class _CallableArgument:
```

Comment:
Regarding the callback (is it Callable or Callback ?), could we move the "callback" to "before we instantiate a pylinter ? I wouldn't mind just having a long list of
```
if args.help:
    print_help()
if args.version:
    print_version()
...
PyLinter(...).run(args)
```
I remember that it was really messy and unnecessary, except maybe for init-hook.

Reply:
No, we need to the `PyLinter` for stuff like ``list-messages`` and ``--help-msg``.

I have called the `Argument` `Callable` since they are callable arguments. I have named the actions `Callback` as that was the terminology previously used.